pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/codestyle/FinalParameterInAbstractMethod.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.codestyle;
+
+import net.sourceforge.pmd.testframework.PmdRuleTst;
+
+public class FinalParameterInAbstractMethod extends PmdRuleTst {
```

Comment:
This should be named `FinalParameterInAbstractMethodTest` (Test at the end)

Reply:
Arg ... wrong copy past. Sorry